test/math/kd_tree_test.cc

```diff
@@ -198,15 +296,30 @@ TEST_F(KDTreeTest, NNSearch) {
   EXPECT_DOUBLE_EQ(expected_point.z(), nearest_point_with_tol.z());
 }
 
+// Evaluates KDTree::Nearest method.
+TEST_F(KDTreeTest, RangeSearch) {
```

Comment:
Correct me if I'm wrong, but in this test case we are evaluating that all points are returned. If so, can we name it accordingly? --> RangeSearchReturnsAllPointsWithEnclosingRegion or something similar.

Reply:
Done.